go/inst/instance_dao.go

```diff
@@ -1139,7 +1139,7 @@ func readInstanceRow(m sqlutils.RowMap) *Instance {
 	instance.SemiSyncEnforced = m.GetBool("semi_sync_enforced")
 	instance.SemiSyncAvailable = m.GetBool("semi_sync_available")
 	instance.SemiSyncMasterEnabled = m.GetBool("semi_sync_master_enabled")
-	instance.SemiSyncMasterTimeout = m.GetUint("semi_sync_master_timeout")
+	instance.SemiSyncMasterTimeout = m.GetInt64("semi_sync_master_timeout")
```

Comment:
```suggestion
	instance.SemiSyncMasterTimeout = m.GetUint64("semi_sync_master_timeout")
```

Reply:
there is no `GetUint64` in https://github.com/openark/golib/blob/master/sqlutils/sqlutils.go